segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/Attributes.java

```diff
@@ -23,8 +23,13 @@
  * MetadataCheckpoints or the Segment's State, thus its value will not be reloaded into memory upon a failover or
  * segment eviction + re-activation.
  */
-public final class Attributes {
+public class Attributes {
     /**
+     * The Attribute ID at which Table Attributes can begin at. Everything with an ID smaller than this is a general
+     * Attribute.
+     */
+    public static final long TABLE_ATTRIBUTES_START_OFFSET = 1024; // Powers of 2 make UUID.toString look readable.
```

Comment:
Should this be larger?

Reply:
I doubt we'll ever have more than 1024 core attributes (that would be a lot!). In any case, I made this 2^20 to future-proof it.